docs/docs/getting-started/demo/with-docker.md

```diff
@@ -10,23 +10,23 @@ import TerminalWindow from '@site/src/components/TerminalWindow';
 
 > Our standard demo setup for demonstrating the Hydra Head protocol.
 
-The demo consists of
+The demo consists of:
 
-- a cluster of three Hydra nodes, directly connected to each other, each having access to one of three Hydra credentials `alice`, `bob` or `carol`;
-- a single Cardano node producing blocks which is used as a local devnet;
-- a prometheus server gathering metrics;
-- ad-hoc terminal user-interface clients to interact with the individual Hydra nodes;
+- a cluster of three Hydra nodes, directly connected to each other, each having access to one of three Hydra credentials `alice`, `bob`, or `carol`;
+- a single Cardano node producing blocks used as a local devnet;
+- a Prometheus server to gather metrics;
+- ad-hoc terminal user interface clients to interact with the individual Hydra nodes;
 
-:::caution
-As we are using a ad-hoc private devnets which start from the genesis block, you need to ensure the devnet configuration is reasonably up-to-date. If you get `LedgerNoView` errors from the Cardano node, that means the start times are too far in the past and you should update them e.g. using the `prepare-devnet.sh` script.
+:::Caution!
```

Comment:
```suggestion
:::caution Caution!
```
This won't work! `caution` here is a keyword recognized by the tool we use to pick the right icon and colors for this note. The title for the note can be adjusted as shown however.